.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push]
```

Comment:
Do we want to limit the linting to just PRs?

Reply:
fixed